### PR TITLE
masstransit interface rework

### PR DESCRIPTION
**Pull request by viktor-kulkov**

### CODE REVIEW

File: src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+namespace AppForeach.Framework.MassTransit;
+
+public class ConsumerConfigurationBuilder<TConsumer> : IConsumerConfigurationBuilder<TConsumer>
+    where TConsumer : class
+{
+    private readonly List<Action<IConsumerConfigurator<TConsumer>>> actions = [];
+
+    public IConsumerConfigurationBuilder<TConsumer> Configure(Action<IConsumerConfigurator<TConsumer>> action)
+    {
+        actions.Add(action);
```
Comment (valdisz) [inline src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs:12]: Check for `null` unless you have #nullable enabled

File: src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        return this;
+    }
+
+    public void ConfigureAll(IConsumerConfigurator<TConsumer> cfg)
+    {
+        foreach (var action in actions) 
+        {
+            action(cfg);
```
Comment (valdisz) [inline src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs:20]: `cfg` can be `null`

**Review by valdisz — APPROVED** [review]

File: src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs (hunk 2 above)
Comment (viktor-kulkov) [reply src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs:24]: fixed

File: src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs (hunk 1 above)
Comment (viktor-kulkov) [reply src/AppForeach.Framework.MassTransit/ConsumerConfigurationBuilder.cs:14]: fixed

**Pull request merged**